src/app/features/ControlPanel/Services/RunnerService.js

```diff
@@ -10,8 +10,11 @@ var RunnerService = function RunnerService (
     notifierService,
     realTimeService
 ) {
+    var configEnv; 

```

Comment:
It's a bit weird that this is stored here and then passed back into this service from the controllers. It would make more sense to update this service from the controller when the drop down changes and then just use the value from within the service, i.e.:

```
function runProtractor (options) {
    options.baseUrl = this.environment;
}
```